package.json

```diff
@@ -39,7 +39,7 @@
     "typeface-asap": "0.0.54",
     "uuid": "^3.3.2",
     "webrtc-swarm": "git+https://github.com/QwantResearch/webrtc-swarm#masq",
-    "masq-common": "git+https://github.com/QwantResearch/masq-common"
+    "masq-common": "git+https://github.com/QwantResearch/masq-common#MASQ-118-MK-protection"
```

Comment:
don't forget to use the master branch now

Reply:
done